### PR TITLE
pass the additional to sign list to the installerspec generator

### PR DESCRIPTION
**Pull request by mjkkirschner**
add new files
add comments
remove files that no longer exist
update installerSpec.exe
update installerspec source

### Purpose

This PR updates the install.sln which is still run during the build to sign binaries - to pass a file which lists additional binaries to sign which might have different version numbers from dynamoCore and thus are skipped in the first pass.

It also lets us add comments to this file, and white space lines....

Additionally, if a file fails to be found which was requested in this list, the build fails.

To find the files we were missing I used SigCheck
https://docs.microsoft.com/en-us/sysinternals/downloads/sigcheck
`sigcheck -u -e -s DynamoCoreRuntime2.6xxx/ > unsigned.txt`

I've attached the list to the jira task, we may want to consider running this over every build in our binary diff job and making sure the number does not change from build to build without us knowing.

I have tested manually and the generated binariestosign.txt includes the files.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers
@QilongTang 
@zeusongit 

### FYIs

@minjie1980

### CODE REVIEW

**Comment by QilongTang** [comment]
Reviewing now

File: tools/InstallerSpec/InstallerSpec/Program.cs [hunk 1]
```diff
@@ -213,22 +216,35 @@ static void Main(string[] args)
...
                                     // contains fully qualified paths instead of doing the prefix here.
                                     // 
                                     var fileNameWithoutPath = streamReader.ReadLine().Trim();
+                                    //this is a comment, ignore this line
+                                    if (fileNameWithoutPath.StartsWith("##") || string.IsNullOrWhiteSpace(fileNameWithoutPath))
+                                    {
+                                        continue;
+                                    }
                                     var fileNameWithPath = Path.Combine(binpath, fileNameWithoutPath);
                                     if (!string.IsNullOrEmpty(fileNameWithPath))
```
Comment (QilongTang) [inline tools/InstallerSpec/InstallerSpec/Program.cs]: Might be too much to ask.. Can we file a follow up task to include folder pattern? I would like to add .\libG* or something like that for future bullet proof.

**Comment by QilongTang** [comment]
@mjkkirschner This LGTM but I would love to add one follow up to make it bullet proof for libG updates

File: tools/InstallerSpec/InstallerSpec/Program.cs (hunk 1 above)
Comment (mjkkirschner) [reply tools/InstallerSpec/InstallerSpec/Program.cs]: I looked into that, .net does not have any glob pattern support - they instead have a nuget package you can reference: 
https://www.nuget.org/packages/Microsoft.Extensions.FileSystemGlobbing/

I didn't want to add additional deps for this now.

Comment (QilongTang) [reply tools/InstallerSpec/InstallerSpec/Program.cs]: @mjkkirschner Yes, I think this PR is good in the current state. Let's do the improvement later.

**Pull request merged**